src/aquarium.py

```diff
@@ -109,49 +109,45 @@ async def aquarium_shutdown(aquarium_app: FastAPI, aquarium_api: FastAPI):
 
 
 def aquarium_factory(
-    startup_method=aquarium_startup,
-    shutdown_method=aquarium_shutdown,
-    static_dir=None
+    startup_method=aquarium_startup, shutdown_method=aquarium_shutdown, static_dir=None
```

Comment:
and this also makes readability worse than having a giant line with all the options there.

Reply:
We could lower the line length which might solve this.

Reply:
I'd rather not argue about line length ;)